packages/dashboard/src/app/views/shared/bodyViewOptions.js

```diff
@@ -91,6 +98,36 @@ export default function BodyViewOptions({
           <TranslateWithMarkup>{resultsLabel}</TranslateWithMarkup>
         </Text>
         <ControlsContainer>
+          {layoutStyle === VIEW_STYLE.GRID && filters?.length
+            ? filters.map((filter) => (
+                <StorySortDropdownContainer key={filter.key}>
+                  <StyledDatalist
+                    hasSearch
+                    hasDropDownBorder
+                    searchResultsLabel={__(
+                      filter.labels.searchItems,
```

Comment:
@spacedmonkey @swissspidy - These filters have to be created programmatically, it seems like `i18n-no-variables` needs these value to be hardcoded though.  Do you have any suggestions on how we should handle these?  (Remove i18n function calls here?) - https://github.com/GoogleForCreators/web-stories-wp/pull/11526#issuecomment-1132880329

Reply:
I think you could use something like this (permalink is to my PR of an example within the Shopping feature, but should be a helpful example):
https://github.com/GoogleForCreators/web-stories-wp/blob/7ac2f5196e00204a7f85a7324b106beaea5ed6a3/packages/story-editor/src/components/library/panes/shopping/productButton.js#L86-89

As far as which line to use, I'd love feedback on that. I'm not sure if searchItems or all-items is the preferred title for the dropdown, sounds like searchItems perhaps.

Reply:
@mwritter I think there's two options here:
1. We leave things the way they are in this PR (but add an override to the linter in this case)
2. Or, we could do something like `Search %s` and the label we use instead is `filter.label.all-items`.

Both aren't the most ideal option, unfortunately, but the second one might allow for more translation opportunities.